app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    new_one = command.split()
+    if len(new_one) != 3 and new_one[0] != "mv":
+        raise ValueError("Invalid command format")
+
+    move, first, second = new_one
+    destination = os.path.dirname(second)
+
+    if destination:
+        os.makedirs(destination, exist_ok=True)
+
+    new_file = os.path.join(destination, os.path.basename(second))
```

Comment:
It seems illogical, you split the variable `second`  to then combine it. Why can't you use `second`?